js/application.js

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
I meant that please fix the indentations for the rest of the file. The
first line shouldn't have indentations

On Tue, Jul 14, 2015 at 11:38 AM Rachel <notifications@github.com> wrote:

> In js/application.js
> <https://github.com/WDI-HK-8/UrbanFictionary/pull/1#discussion_r34534995>:
>
> > @@ -0,0 +1,16 @@
> > +
>
> Oh I thought there was no indentation for first line - okay, indented.
> Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WDI-HK-8/UrbanFictionary/pull/1/files#r34534995>.
>